src/main.js

```diff
@@ -109,9 +109,9 @@ class MiniMediaPlayer extends LitElement {
       toggle_power: true,
       ...config,
       hide: { ...DEFAULT_HIDE, ...config.hide },
-      sonos: {
+      speaker_group: {
         show_group_count: true,
-        ...config.sonos,
+        ...config.speaker_group,
```

Comment:
```js
...config.sonos,
...config.speaker_group,
```
to avoid breaking change, what do you think?

Reply:
Makes perfect sense. added in new commit